NextcloudTalk/ContactsTableViewCell.m

```diff
@@ -95,4 +98,19 @@ - (void)setUserStatus:(NSString *)userStatus
     }
 }
 
+- (void)setUSerStatusMessage:(NSString *)userStatusMessage withIcon:(NSString*)userStatusIcon {
+    if ((userStatusMessage && ![userStatusMessage isEqualToString:@""] && userStatusMessage != NULL)) {
+        if (userStatusIcon && ![userStatusIcon isEqualToString:@""] && userStatusIcon != NULL) {
```

Comment:
No need to check for` != NULL` since we already check `if ((userStatusMessage` and `if ((userStatusIcon`